runtime/runtime.go

```diff
@@ -634,21 +662,27 @@ func (r *interpreterRuntime) injectedCompositeFieldsHandler(
 		compositeKind common.CompositeKind,
 	) map[string]interpreter.Value {
 
-		switch compositeKind {
-		case common.CompositeKindContract:
-			var address []byte
+		switch location {
+		case stdlib.CryptoChecker.Location:
```

Comment:
Do you have ideas for how to standardize these kinds of operations once more standard libraries are added? I'm just a bit nervous about having a special case in all of these switch statements.

Reply:
It would be nice to make this more declarative, similar to / in `FlowBuiltinImpls`, but as noted below I don't have good ideas yet. If you do, please let me know!